src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -38,7 +38,10 @@ export default function VitalPercents(props: Props) {
                       })
                 }
               >
-                <IconFire color={vitalStateColors[p.vitalState] as Color} />
+                <IconFire
+                  style={{verticalAlign: 'middle'}}
```

Comment:
Instead of inline styles you can use `css` from `@emotion/core` and apply it to each of these Icons using the `css` prop.

Reply:
I totally rewrote this in v3 of this PR. Could you possibly review it again? Thanks!

Reply:
Oh, was I looking at an old commit or something? My bad